package-lock.json

```diff
@@ -67558,8 +67558,7 @@
 					"integrity": "sha512-GUy/3ZY4kW1KdYHtMZRrRlj5FYbZTAyHlimxHI7Zs0xsC+kAzIf8lopnf67Y9IYLgEMr37KosIV7kwpkJpNs5Q=="
 				},
 				"@ariakit/react": {
-					"version": "0.4.7",
-					"resolved": "https://registry.npmjs.org/@ariakit/react/-/react-0.4.7.tgz",
+					"version": "https://registry.npmjs.org/@ariakit/react/-/react-0.4.7.tgz",
```

Comment:
Is this supposed to be correct? The new version of `package-lock.json` still points at a `0.4.7` file

Reply:
Nope, this isn't right, I'll take a look

Reply:
Should be resolved by b937a27cb580d5eb63b462a97adbe8fa6503ddf6. 

There was a dependency duplication going on, and I addressed it.